packages/editor/src/components/provider/index.native.js

```diff
@@ -242,6 +253,7 @@ export default compose( [
 			getEditedPostContent,
 			selectedBlockIndex: getBlockIndex( selectedBlockClientId ),
 			blockCount: getBlockCount( rootClientId ),
```

Comment:
Hey @SergioEstevao , I know that this line is not part of the PR but, now that we import `getGlobalBlockCount` what do you think about computing the block count using `getGlobalBlockCount()` [like here](https://github.com/WordPress/gutenberg/blob/73a79f177194d954805d80da2b4705c06a3e5f20/packages/editor/src/components/table-of-contents/panel.js#L80)?

Reply:
Ok, I will update the code for that.